portal/models/assessment_status.py

```diff
@@ -105,7 +105,7 @@ class QuestionnaireBankDetails(object):
     """
     def __init__(self, user):
         self.user = user
-        self.qb = QuestionnaireBank.most_current_qb(user)
+        self.qb = QuestionnaireBank.most_current_qb(user)[0]
```

Comment:
More "pythonic" style is to throw away the part of the return tuple you don't need:

`self.qb, _ = QuestionnaireBank.most_current_qb(user)`

Reply:
@pbugni Gotcha. Updated all existing calls this way.